src/Umbraco.Core/Logging/DisposableTimer.cs

```diff
@@ -85,19 +118,55 @@ protected override void DisposeResources()
             {
                 if (_failed)
                 {
-                    _logger.LogError(_failException, "{FailMessage} ({Duration}ms) [Timing {TimingId}]", _failMessage, Stopwatch.ElapsedMilliseconds, _timingId);
+                    if (_failMessageArgs == null)
```

Comment:
Very minor thing but I think that `is null` is more readable that `== null`, however looking it up the Roslyn compiler has been updated so the performance is the same